cmd/eksctl-anywhere/cmd/createcluster.go

```diff
@@ -114,24 +145,6 @@ func (cc *createClusterOptions) createCluster(cmd *cobra.Command) error {
 		return fmt.Errorf("Error: provider tinkerbell is not supported in this release")
 	}
 
-	if deps.Provider.Name() == "tinkerbell" {
-		flag := cmd.Flags().Lookup("hardwarefile")
-		if flag == nil {
-			return fmt.Errorf("Something wrong. Flag hardwarefile not set up for provider tinkerbell")
-		}
-		if !viper.IsSet("hardwarefile") || viper.GetString("hardwarefile") == "" {
-			return fmt.Errorf("Error: required flag \"hardwarefile\" not set")
-		}
-		hardwareConfigFileExist := validations.FileExists(cc.hardwareFileName)
-		if !hardwareConfigFileExist {
-			return fmt.Errorf("Error: hardware config file %s does not exist", cc.hardwareFileName)
-		}
-	}
-
-	if !features.IsActive(features.CloudStackProvider()) && deps.Provider.Name() == constants.CloudStackProviderName {
-		return fmt.Errorf("error: provider cloudstack is not supported in this release")
-	}
```

Comment:
why remove this?

Reply:
Oops. Looks like a merge mistake, my bad. Will fix in the morning.

Reply:
Fixed